contributors/devel/sig-testing/monitoring.md

```diff
@@ -0,0 +1,264 @@
+
+---
+
+## What do I do when I see a TestGrid alert?
+
+If you are part of a SIG's mailing list, occasionally you may see emails from
+TestGrid reporting that a job or a test has recently failed. If you are browsing
+through TestGrid, you may also see jobs labeled as "flaky" (in purple) or as
+"failing" (in red). This section is to help guide you on what to do in these
+occasions.
+
+### Communicate your findings
+
+The number one thing to do is to communicate your findings: a test or job has
+been flaking or failing. If you saw a TestGrid alert on a mailing list, please
+reply to the thread and mention that you are looking into it.
+
+It is important to communicate to prevent duplicate work. First, check GitHub to
+see if an issue has already been logged by checking the following ... 
+
+ - [Flaky Tests that need to be
+   triaged](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+kind%2Fflake)
+ - [Triaged Flaky
+   Tests](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+kind%2Fflake+-label%3Aneeds-triage+)
+ - [CI Signal board](https://github.com/orgs/kubernetes/projects/11) which
+   segments logged issues by workflow
+ 
+If an issue has already been opened for the test  you can add any new
+findings that are not already documented in the issue.
+
+For example, if a test is flaking intermittently and you have found another
+incident where the test has failed that has not been recorded in the issue then
+add the new information to the existing issue.
+
+You can:
+
+ - add a link to the Prow job where the latest test failure has occurred. 
+ - note the error message
+  
+New evidence is especially useful if the root cause of the problem with the test
+has not yet been determined and the issue still has a *needs-triage* label.
+
+If the issue has not already been logged please open up an issue on Kubernetes,
+https://github.com/kubernetes/kubernetes/issues/new/choose, and choose the
+appropriate issue template, [Failing Test ](https://github.com/kubernetes/kubernetes/issues/new?assignees=&labels=kind%2Ffailing-test&template=failing-test.md) or [Flaking
+Test](https://github.com/kubernetes/kubernetes/issues/new?assignees=&labels=kind%2Fflake&template=flaking-test.md)
+
+#### Filling out an issues
+Both test issue templates are reasonably self explanatory so what follows here
+are guidelines and tips on filling out the templates.
+
+When logging a Flaking or Failing test please
+ - use plain text when referring to test names, job names. Inconsistent
+   formatting of names makes it harder to process issues via automation.
+ - keep an eye out for test names that contain markdown parse-able formatting.
+   If you are a test maintainer refrain from including markdown in strings that
+   are used to name your tests, test components etc.
+ 
+#### Fill out the issue for a Flaking Test
+
+1. **Which jobs are flaking**
+
+The job is the tab in TestGrid that you are looking at.
```

Comment:
Strongly agree.  The job is _not_ the name of the tab.  It is really important to distinguish the two, because you need the job name to:
- find job config in kubernetes/test-infra
- look at recent runs in https://prow.k8s.io
- look up failures in https://go.k8s.io/triage
- look up top flakes in http://storage.googleapis.com/k8s-metrics/flakes-latest.json

I really think a glossary or explainer up top is needed:
- job name: e.g. `ci-kubernetes-foo`, shows up in prow.k8s.io, is the name of the gcs bucket where results are stored, etc
- testgrid tab name: 'build', is a shortened and possibly non-unique name that is well understand given the context of the dashboard in which it lives

How to translate from one to the other? Quickest way as a human is to click on a cell to see results in prow.k8s.io's spyglass, it will list the job name up top

Reply:
Should we report this as a User Ex1perience bug in TestGrid? 

What is the justification for having having the tab name be different from the job name? Seems needlessly confusing to me. 

Also Jorge called this UX pothole out towards the end of the doc.